actors/builtin/multisig/multisig_actor.go

```diff
@@ -69,12 +72,7 @@ type ProposeParams struct {
 	Params abi.MethodParams
 }
 
-type ProposeReturn struct {
-	TxnID TxnID
-}
-
-func (a *MultiSigActor) Propose(rt vmr.Runtime, params *ProposeParams) *ProposeReturn {
-	rt.ValidateImmediateCallerType(builtin.CallerTypesSignable...)
+func (a *MultiSigActor) Propose(rt vmr.Runtime, params *ProposeParams) *cbg.CborInt {
```

Comment:
Why is this necessary? `ProposeReturn` was already CBORable.

Reply:
It also serializes as a struct, causing extra wasted bytes, and deviates from existing expectations around return values